### PR TITLE
fix: Accept collaborative close offer from coordinator

### PR DESCRIPTION
**Pull request by holzeis**
I noticed that both parties can trigger a collaborative dlc channel close, but only the coordinator had the logic implemented to also accept it. This will allow the app also to accept a collab close offer from the coordinator.

### CODE REVIEW

**Review by bonomat — COMMENTED** [review]
Afaik @luckysori also added this in one of his branches. 
Let's wait for his approval before merging please.

**Comment by luckysori** [comment]
> Afaik @luckysori also added this in one of his branches. Let's wait for his approval before merging please.

I don't think I added this logic :D

**Review by luckysori — APPROVED** [review]

**Pull request merged**
